ArmPkg/Drivers/MmCommunicationDxe/MmCommunication.c

```diff
@@ -259,8 +259,12 @@ GetMmCompatibility (
 
   MmVersion = MmVersionArgs.Arg0;
 
+  STATIC_ASSERT (
+    MM_CALLER_MINOR_VER == 0,
+    "MM_CALLER_MINOR_VERSION has changed!"
+    );
   if ((MM_MAJOR_VER (MmVersion) == MM_CALLER_MAJOR_VER) &&
-      (MM_MINOR_VER (MmVersion) >= MM_CALLER_MINOR_VER))
+      (MM_MINOR_VER (MmVersion) == MM_CALLER_MINOR_VER))

```

Comment:
While this may have been flagged by CodeQL, this is a behavior change, so I think it should at a minimum come in a separate commit. I also remember this change being made, but I think it should be revisited, I think the existing behavior of allowing a greater minor version from StMM makes sense (although perhaps this could be `!=` instead of only allowing the StMM side to be a greater minor version).

Reply:
Just realised I missed posting my review comments on Friday.